app/src/main/java/com/celzero/bravedns/ui/HomeScreenActivity.kt

```diff
@@ -243,12 +266,13 @@ class HomeScreenActivity : AppCompatActivity(R.layout.activity_home_screen) {
     }
 
     private fun isGooglePlayServicesAvailable(): Boolean {
-        val googleApiAvailability: GoogleApiAvailability = GoogleApiAvailability.getInstance()
-        val status: Int = googleApiAvailability.isGooglePlayServicesAvailable(this)
-        if (status != ConnectionResult.SUCCESS) {
-            return false
+        return try {
+            // applicationInfo.enabled - When false, indicates that all components within
+            // this application are considered disabled, regardless of their individually set enabled status.
+            packageManager.getApplicationInfo(PLAY_SERVICE_PKG_NAME, 0).enabled
```

Comment:
Add a `TODO` that we must inform the user via dialog that PlayServices was disabled and if they would like to switch to website version for updates.

Reply:
added.